src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -4,44 +4,56 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.nullValue;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 
-import java.net.MalformedURLException;
-import java.util.UUID;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeoutException;
-
+import org.folio.circulation.support.ClockManager;
+import org.folio.circulation.support.JsonPropertyFetcher;
 import org.folio.circulation.support.http.client.IndividualResource;
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
-import org.junit.Test;
 
 import api.support.APITestContext;
 import api.support.APITests;
 import api.support.http.OkapiHeaders;
 import io.vertx.core.json.JsonObject;
+import java.util.UUID;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
```

Comment:
Why have these imports moved to here? I don't think that fits with our [provisionally agreed import ordering|https://issues.folio.org/browse/FOLIO-1667]

Reply:
The imports have been reformatted with the folio.importoder.

Reply:
Has that been done for all changed files, or only this one?

Reply:
All of the changed files.

Reply:
Thank you